2023-round-1/navyagarwal/nx_tutorial_script.py

```diff
@@ -0,0 +1,49 @@
+# imports
+import networkx as nx
+import matplotlib.pyplot as plt
+
+# DiGraph -> Direted graph that can contain self-loops but not parallel edges
+G = nx.DiGraph()
+
+# various methods to add nodes of different data types
+G.add_node(1)                      # integer
+G.add_node('navya')                # string
+G.add_node((3, 4))                 # tuple
+G.add_nodes_from(['a', 'b'])       # from list
+G.add_nodes_from(range(9, 11))     # in range
+
+X = nx.path_graph(2)
+Y = nx.complete_graph(3)
+
+G.add_nodes_from(X)                # from graph X
+G.add_node(Y)                      # graph Y
+
+
```

Comment:
Any specific reason for doing this?

Wouldn't this just end up adding the node 0, 1, 2 again (which won't really make any difference to the Graph `G`)